src/infrastructure/health/notification.check.ts

```
@@ -33,12 +71,40 @@ export class NotificationCheck {
+      connectTimeout: 1000,
+      retryStrategy: (times) => Math.min(times * 100, 1000),
     });
+    this.redis.on('error', () => undefined);
+
+    return this.redis;
+  }
+
+  private hasRedisConfiguration() {
+    return Boolean(
+      this.configService.get<string>('ENDPOINT_HOST') &&
+        this.configService.get<string>('ENDPOINT_PORT'),
```

ESLint

[error] Delete `··`

(prettier/prettier)